rapids-cmake/export/export.cmake

```diff
@@ -182,9 +185,9 @@ function(rapids_export type project_name)
     rapids_export_parse_version(${_RAPIDS_VERSION} rapids_orig rapids_project_version)
   endif()
 
-  set(_RAPIDS_PROJECT_NAMESPACE "${project_name}::")
-  if(DEFINED _RAPIDS_NAMESPACE)
-    set(_RAPIDS_PROJECT_NAMESPACE ${_RAPIDS_NAMESPACE})
+  if(NOT DEFINED _RAPIDS_NAMESPACE)
+    message(NOTICE "rapids-cmake EXPORT: no NAMESPACE was provided. `${project_name}::` is recommended \
```

Comment:
I think this should be a `WARNING`  or maybe `VERBOSE`. We haven't used `NOTICE` before in rapids-cmake as that outputs in the error style and this use could be intentional.

Reply:
I don't love the `WARNING` since users could have done that on purpose and used one directly with `EXPORT_NAME`. `VERBOSE` then?

Reply:
Lets go with `VERBOSE`